src/date-picker/date-picker.jsx

```diff
@@ -149,11 +149,14 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: (date) => {
```

Comment:
I just moved it into a function, but still gets shown in the docs as: 

function toIsoDateString(date) { return (new Date(`${date.toDateString()} 12:00:00 +0000`)).toISOString().substring(0, 10);}